src/Websites/Websites/Cmdlets/WebApps/PublishAzureWebApp.cs

```diff
@@ -45,6 +45,9 @@ public class PublishAzureWebAppCmdlet : WebAppOptionalSlotBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Sets the timespan in Milliseconds to wait before the request times out.")]
+        public double Timeout { get; set; }
```

Comment:
You can rely on PowerShell to limit the ranage of a parameter
```suggestion
        [ValidateRange(100000, double.MaxValue)]
        public double Timeout { get; set; }
```

Reply:
Thanks for your suggestion. made it as suggested.